content/applications/sales/sales/send_quotations/create_quotations.rst

```diff
@@ -0,0 +1,332 @@
+fields that can be configured.
+
+.. image:: create_quotations/other-info-invoicing.png
+   :align: center
+   :alt: The Invoicing section of the Other Info tab of a quotation form in Odoo Sales.
+
+- :guilabel:`Fiscal Position`: select a fiscal position to be used in order to adapt taxes and
+  accounts for particular customers or sales orders/invoices. The default value comes from the
+  customer. If a selection is made in this field, an :icon:`fa-refresh` :guilabel:`Update Taxes`
+  clickable link and icon appear. When clicked, the taxes for this partiuclar customer and quotation
+  are updated.
+- :guilabel:`Analytic Account`: select an analytic account to apply to this customer/quotation.
+
+Tracking section
+~~~~~~~~~~~~~~~~
+
+In the :guilabel:`Tracking` section of the :guilabel:`Other Info` tab, there are tracking specific
+fields that can be configured.
+
+.. image:: create_quotations/other-info-tracking.png
+   :align: center
+   :alt: The Tracking section of the Other Info tab of a quotation form in Odoo Sales.
+
+- :guilabel:`Source Document`: enter the reference of teh document that generated the
+  quotation/sales order, if applicable.
+- :guilabel:`Opportunity`: select the specific opportunity (from the *CRM* app) related to this
+  quotation, if applicable.
+- :guilabel:`Campaign`: select the marketing campaign related to this quotation, if applicable.
+- :guilabel:`Medium`: select the method of which this quotation was originated (e.g. *Email*,
+  *Facebook*, etc.), if applicable.
+- :guilabel:`Source`: select the source of the link used to generate this quotation, if applicable.
+
+Notes tab
+---------
+
+In the :guilabel:`Notes` tab of the quotation form, choose to enter any specific internal notes
+about the quotation and/or customer, if desired.
+
+Sending and confirming quotations
+=================================
+
+Once all the necessary fields and tabs have been configured on the quotation form, it is time to
+send the quotation to the customer, so they can confirm it, which subsequently, turns the quotation
+into an official sales order.
+
+At the top of the form, there is a series of buttons:
+
+- :guilabel:`Send by Email`: when clicked, a pop-up window appears with the customer's name and
+  email address in the :guilabel:`Recipients` field, the quotation (and reference ID) in the
+  :guilabel:`Subject` field, and a brief default message in the body of the email, which can be
+  modified, if needed.
+
+  Below that, a PDF copy of the quotation is attached. When ready, click :guilabel:`Send` to send
+  the quotation, via email, to the customer, so they can review and confirm it.
+- :guilabel:`Send PRO-FORMA Invoice`: this button **only** appears if the *Pro-Forma Invoice*
+  setting has been enabled. When clicked, a pop-up window appears with the customer's name and email
+  address in the :guilabel:`Recipients` field, the *Proforma* invoice (and reference ID) in the
```

Comment:
"Proforma" is formatted differently here than above.

Reply:
For sure, and I would've mentioned the same thing. However, like some other aspects of the U/I, the spelling/formatting isn't always the same throughout the database, unfortunately. So, with that in mind, I always opt for the formatting/spelling that is present on the screen *at the moment*.

In this case, it is spelled/formatted like this one the pop-up:
![Screenshot 2024-06-13 105142](https://github.com/odoo/documentation/assets/73958186/627e73d3-185d-4524-bfd3-54b88478e444)